File: fix_server.js
const fs = require('fs');
let code = fs.readFileSync('server.ts', 'utf-8');
code = code.replace("không bọc trong markdown code block (```).", "không bọc trong markdown code block (\\\`\\\`\\\`).");
fs.writeFileSync('server.ts', code);

File: fix_server.py
import sys

with open("server.ts", "r") as f:
    content = f.read()

# Replace all broken variations
content = content.replace("không bọc trong markdown code block (\\\\`\\\\`\\\\`).", "không bọc trong markdown code block (\\`\\`\\`).")
content = content.replace("không bọc trong markdown code block (```).", "không bọc trong markdown code block (\\`\\`\\`).")

with open("server.ts", "w") as f:
    f.write(content)
